packages/react-day-picker/src/components/Dropdown/Dropdown.tsx

```diff
@@ -3,13 +3,19 @@ import React from 'react';
 import { IconDropdown } from 'components/IconDropdown';
 import { useDayPicker } from 'contexts/DayPicker';
 
+type DropdownOption = {
+  name: string;
```

Comment:
- This type should be exported and have some comment like the interface below.
- What is `name` used for? Maybe you mean label?

```suggestion
  label: string;
```

Reply:
Yeah, it was meant to be the label of the value